boards/arm/nrf52_vbluno52/doc/nrf52_vbluno52.rst

```diff
@@ -0,0 +1,109 @@
+.. _nrf52_vbluno52:
+
+nRF52-VBLUno52
+##############
+
+Overview
+********
+
+Zephyr can use the nrf52_vbluno52 board configuration to run on the VBLUno52 board,
+a VNG Bluetooth Low Energy UNO using an nRF52832 ARM Cortex-M4F processor.
+
+.. figure:: img/nrf52_vbluno52_pinout.png
+     :align: center
+     :alt: nRF52 VBLUno52
+
+     nRF52_VBLUno52 board
+
+Hardware
+********
+
+The VBLUno52 board has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 64 MHz.
+
+Supported Features
+==================
+
+- CPU:
+	+ Nordic nRF52832: ARM® Cortex™ M4F 32bit.
+	+ *Bluetooth Low Energy 5.0 interface.*
+	+ 512KB Flash, 64KB Ram.
+	+ UART(1), I2C(2), SPI(3), PWM(3), SWD, I2S, Timer 32bit(5), RTC(3).
+	+ Support EasyDMA.
+	+ NFC Tag.
+	+ 21 digital channels, 6 ADC 10bit channels.
+	+ 1 Led and 1 Button onboard.
+	+ GPIO Voltage: 0 – 3.3V.
+- *DAPLink (CMSIS-DAP) interface for program and debug*:
+	+ USB MSD: Drag and Drop programming flash memory.
+	+ USB HID (DAP): CMSIS-DAP compliant debug channel.
+	+ USB CDC: Virtual COM port for log, trace and terminal emulation.
+- Supports hardware flow control features (RTS/CTS).
+- *Energy monitoring for BLE module by current measurement*
+- FOTA (Firmware over the air): Upgrade firmware over BLE interface.
+- Build good applications with:
```

Comment:
Simililar question.  This looks like a cut/paste of generic board information, but we should focus on what Zephyr supports (and leave our discussion about other OS support such as Arduino, RIOT-OS, etc.)